# Contiene el angulo minimo y maximo de covertura del objeto y el theta de la al punto mas cercano del objeto
float32 min_theta
float32 theta
float32 max_theta

# Contiene la norma minimo y maximo de covertura del objeto y el theta de la al punto mas cercano del objeto
float32 min_norm
float32 norm
float32 max_norm
